ghedesigner/radial_numerical_borehole.py

```diff
@@ -249,22 +218,20 @@ def fill_radial_cell(self, radial_cell, resist_p_eq, resist_f_eq, resist_tg_eq):
                 ],
                 dtype=np.double,
             )
-        cell_summation += num_soil_cells
+        cell_summation += self.num_soil_cells
 
     def calc_sts_g_functions(self, single_u_tube, final_time=None) -> tuple:
 
         self.partial_init(single_u_tube)
 
         resist_bh_effective = self.single_u_tube.calc_effective_borehole_resistance()
-
         resist_f_eq = self.single_u_tube.R_f / 2.0
-        resist_p_eq = self.single_u_tube.R_p / 2.0
-        resist_tg_eq = resist_bh_effective - resist_f_eq
+        resist_pg_eq = resist_bh_effective - resist_f_eq
 
         # Pass radial cell by reference and fill here so that it can be
         # destroyed when this method returns
         radial_cell = np.zeros(shape=(len(CellProps), self.num_cells), dtype=np.double)
-        self.fill_radial_cell(radial_cell, resist_p_eq, resist_f_eq, resist_tg_eq)
+        self.fill_radial_cell(radial_cell, resist_f_eq, resist_pg_eq)
```

Comment:
Previously, the pipe, fluid, and effective pipe-grout resistance values were being passed in and used separately. As it stand presently, only the fluid and pipe-grout resistances are needed, so this extra argument was removed.